pkg/commands/compute/deploy.go

```diff
@@ -839,37 +837,43 @@ func pkgUpload(spinner text.Spinner, client api.Interface, serviceID string, ver
 	return spinner.Stop()
 }
 
+// setupObjects is a collection of backend objects created during setup.
+// Objects may be nil.
+type setupObjects struct {
+	domains      *setup.Domains
+	backends     *setup.Backends
+	dictionaries *setup.Dictionaries
+	loggers      *setup.Loggers
+	kvStores     *setup.KVStores
+	secretStores *setup.SecretStores
+}
+
 func constructSetupKVs(
```

Comment:
See note: https://github.com/fastly/cli/pull/904/files#r1164377390

Reply:
Agreed this should be `constructSetupObjects`